Gemfile

```diff
@@ -9,7 +9,7 @@ gem "logging", "2.1.0"
 gem 'nokogiri', "1.6.7.2"
 gem 'whenever', require: false
 gem "slop", "3.4.5"
-gem "sidekiq", "< 3"
+gem "sidekiq", "< 4"
```

Comment:
When this PR goes in, another should be raised to upgrade `sidekiq` again, as :
The `< X` is recommended, see:

https://github.com/mperham/sidekiq/blob/master/3.0-Upgrade.md
https://github.com/mperham/sidekiq/blob/master/4.0-Upgrade.md

I think when the last sidekiq bump goes in, it should lock the version